fabrictestbed_extensions/fablib/node.py

```diff
@@ -2735,3 +2749,333 @@ def add_fabnet(
                     subnet=self.get_fablib_manager().FABNETV6_SUBNET,
                     next_hop=net.get_gateway(),
                 )
+
+    def poa(
+        self,
+        operation: str,
+        vcpu_cpu_map: List[Dict[str, str]] = None,
+        node_set: List[str] = None,
+        keys: List[str] = None,
+    ) -> Union[Dict, str]:
```

Comment:
Should `poa()` be a private method? From what I understood by reading the notebooks, `poa()` appears to be a low-level building block.

Reply:
It is intentionally left public to enable users to invoke POAs directly (e.g: add/remove ssh keys) or do pinning in a way different than the default implementation